elfo-logger/src/actor.rs

```diff
@@ -104,6 +107,7 @@ impl Logger {
                             file = open_file(self.ctx.config()).await;
                             use_colors = can_use_colors(self.ctx.config());
                             self.filtering_layer.configure(&self.ctx.config().targets);
+                            self.buffer.max_line_size = self.ctx.config().max_line_size.0 as _;
```

Comment:
`LineBuffer` is not simply structure, it's, actually, an object because it encapsulates some behavior (not complex right now, but more invariants can be easily added in the future).
Thus, any direct access to internals should be forbidden ([LoD](https://en.wikipedia.org/wiki/Law_of_Demeter)).

* `self.buffer.configure(...)` instead of `self.buffer.max_line_size = ...`
* `self.buffer.as_bytes()` instead of `self.buffer.buffer.as_bytes()`

`self.buffer.as_bytes()` can also check (by `(debug_)assert`) implicit invariant: additional buffers (`fields` and `payload`) must be empty by that time.